pkgs/by-name/in/inori/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "inori";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "eshrh";
+    repo = "inori";
+    rev = "v${version}";
+    hash = "sha256-yd1kIlPepVbeTEFzjxTDWEh8o4m6dh/ya9GitqYHHT8=";
+  };
+
+  cargoHash = "sha256-mmzJXIl0wfcyEaYc2wZfA22ZEbTQXG5LVqxEQ4+x76M=";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "inori client for the Music Player Daemon (MPD)";
```

Comment:
```suggestion
    description = "Music Player Daemon (MPD) client built with Rust";
```

Reply:
Using the first line of the upstream's [README.md](https://github.com/eshrh/inori/blob/436873da2df9245679529fb1da8748964fb94186/README.md?plain=1#L22) for the description.